projects/demo-playwright/tests/kit/input-month/input-month.spec.ts

```diff
@@ -0,0 +1,39 @@
+import {TuiDocumentationPagePO, tuiGoto, TuiInputMonthPO} from '@demo-playwright/utils';
+import {expect, Locator, test} from '@playwright/test';
+
+const {describe, beforeEach} = test;
+
+describe(`InputMonth`, () => {
+    describe(`API`, () => {
+        let documentationPage: TuiDocumentationPagePO;
+        let example: Locator;
+        let inputMonth: TuiInputMonthPO;
+
+        test.use({
+            viewport: {width: 360, height: 500},
+        });
+
+        beforeEach(async ({page}) => {
+            await tuiGoto(page, `components/input-month/`);
+            documentationPage = new TuiDocumentationPagePO(page);
+            example = documentationPage.apiPageExample;
+            inputMonth = new TuiInputMonthPO(example.locator(`tui-input-month`));
+        });
+
+        test(`Maximum month less than current month`, async ({page}) => {
+            await tuiGoto(page, `components/input-month/API?tuiMode=null&max$=1`);
+            await inputMonth.textfield.click();
+
+            await documentationPage.prepareApiPageBeforeScreenshot();
+            await expect(page).toHaveScreenshot(`input-month-maximum-month.png`);
```

Comment:
✅ 
![input-month-maximum-month-chromium-darwin](https://github.com/taiga-family/taiga-ui/assets/35179038/5b5e21f0-340d-4ecc-9c7f-9028b42304f4)

Reply:
(but we can decrease the height a little bit)